src/newlines.cpp

```diff
@@ -3231,7 +3231,7 @@ void newlines_chunk_pos(c_token_t chunk_type, tokenpos_e mode)
    chunk_t *prev;
    int     nl_flag;
 
-   if ((mode & (TP_JOIN | TP_LEAD | TP_TRAIL)) == 0)
+   if ((mode & (TP_JOIN | TP_LEAD | TP_TRAIL)) == 0 && chunk_type != CT_COMMA)
```

Comment:
@mihaipopescu Yeep, easy. See changes in uncrustify.cpp:

```cpp
if (cpd.settings[UO_pos_comma].tp != TP_IGNORE || cpd.settings[UO_pos_enum_comma].tp != TP_IGNORE)
{
   newlines_chunk_pos(CT_COMMA, cpd.settings[UO_pos_comma].tp);
}
```

with config

```
pos_comma = ignore
pos_enum_comma = trail_force
```

it will return (because ```(mode & (TP_JOIN | TP_LEAD | TP_TRAIL)) == 0```) and leave enums unchanged.

It has produced expected results for ```pos_class_comma``` independently of ```pos_comma``` value, because class commas are separately handled in  ```newlines_class_colon_pos()```.